.github/stale.yml

```diff
@@ -2,6 +2,10 @@ daysUntilStale: 60
 daysUntilClose: 7
 exemptLabels:
   - "status: accepted"
+  - "type: bug"
```

Comment:
I'm not sure about adding bugs here. 
In my opinion it's a good thing that bugs are marked as stale after a while, because most likely there have been a few changes in the meantime and the bug might not exist anymore. Also, if a reproducable bug is found, we mark it as "accepted", which is exempt anyway